bcdaworker/worker/worker.go

```diff
@@ -233,13 +233,13 @@ func writeBBDataToFile(ctx context.Context, r repository.Repository, bb client.A
 			bene, err := getBeneficiary(ctx, r, uint(id), bb, fetchBBId, jobArgs)
 			if err != nil {
 				//MBI is appended inside file, not printed out to system logs
-				return fmt.Sprintf("Error retrieving BlueButton ID for cclfBeneficiary MBI %s", bene.MBI), fhircodes.IssueTypeCode_NOT_FOUND, err
+				return fmt.Sprintf("Error retrieving BlueButton ID for cclfBeneficiary MBI %s", bene.MBI), fhircodes.IssueTypeCode_PROCESSING, err
```

Comment:
I think this one + the one below it may work better remaining as not_found, since they come directly from another system, and the code returned from BB in these scenarios is a 404. For the ones where the OperationOutcome we return is based upon a BCDA consumer's request (eg the parent job cancelled) then processing is definitely the most appropriate.

Reply:
agreed, I went back and forth on changing this one. Let's leave it as `not_found`; I'll revert this code.